src/cc/CCassetstx_impl.h

```diff
@@ -21,72 +21,95 @@
 #include "CCTokelData.h"
 
 template<class T, class A>
-UniValue AssetOrders(uint256 refassetid, CPubKey pk)
+UniValue AssetOrders(uint256 refassetid, const CPubKey &mypk, const UniValue &params)
```

Comment:
I like to always try to pass const refs as you do here. Be aware that for mypk you are making a copy below, thereby negating the primary reason for const refs (avoiding the copy).

There are other reasons for passing const refs (interface continuity is a popular one), so I am not against it here. I'm just sharing my thoughts.

Also, you may wish to consider
```suggestion
UniValue AssetOrders(uint256 refassetid, const CPubKey &mypk, const UniValue &params = NullUniValue)
```

Which would avoid needing to change things like the `tokenorders` method in tokenrpc.cpp